test/common/crypto/BUILD

```diff
@@ -23,3 +24,10 @@ envoy_cc_test(
         "//source/extensions/common/crypto:utility_lib",
     ],
 )
+
+envoy_cc_fuzz_test(
+    name = "utility_fuzz_test",
```

Comment:
It's a good question. I was planning to rename it, but the original unit test file in the same directory is named as utility_test.cc. If I rename it, it may cause inconsistency. As for the fuzzer with the same name, I noticed this yesterday as well. Sam said as long as they are in different package[bazel path] it should be ok. So I didn't rename it. Do you think it is necessary?

Reply:
We can keep as is, for consistency. It's not difficult to figure out which fuzzer is reporting crashes based on the stack traces. They're all utilities in the end, it's just a personal nit I'm not sure how to deal with.